README.md

```diff
@@ -985,6 +985,22 @@ Usage: lean research [OPTIONS] PROJECT
   By default the official LEAN research image is used. You can override this using the --image option. Alternatively
   you can set the default research image using `lean config set research-image <image>`.
 
+  To expose additional ports to the docker container, update your project's config.json to include the "docker-ports" object.
+
+  For example if you want to expose Tensorboard (defaults to port 6006),
+
+  ...
+  "docker-ports": {
+    "6006": "6006"
+  }
+
+  Likewise for environment variables:
+
+  ...
+  "docker-env": {
+    "PYTHONHASHSEED": "0"
+  }
```

Comment:
This part of the readme is automatically generated by the `scripts/readme.py` script based on the `--help` messages of each command, so the next time that script is ran this piece of documentation will be removed. Project configuration is documented on [https://www.lean.io/docs/lean-cli/user-guides/configuration#04-Project-Configuration](https://www.lean.io/docs/lean-cli/user-guides/configuration#04-Project-Configuration), which is generated from [https://github.com/QuantConnect/Documentation/blob/master/05%20Lean%20CLI/02%20User%20Guides/02%20Configuration/04%20Project%20Configuration.html](https://github.com/QuantConnect/Documentation/blob/master/05%20Lean%20CLI/02%20User%20Guides/02%20Configuration/04%20Project%20Configuration.html). I recommend moving the documentation for `docker-ports` and `docker-env` to that file.